modules/ROOT/pages/changes-deprecations-removals.adoc

```diff
@@ -494,6 +494,9 @@ Replaced by: xref:procedures.adoc#procedure_dbms_cluster_secondaryreplicationdis
 | Name
 | Comment
 
+|xref:monitoring/metrics/reference.adoc#raft-metrics[`<prefix>.cluster.raft.tx_retries`] label:deprecated[Deprecated in 2025.02]
+|
```

Comment:
@Frannie-Ludmilla, can we add a comment here saying that this metric will be removed in future releases?